db/commitlog/commitlog_replayer.cc

```diff
@@ -349,6 +352,9 @@ future<> db::commitlog_replayer::recover(std::vector<sstring> files, sstring fna
                             if (stats.corrupt_bytes != 0) {
                                 rlogger.warn("Corrupted file: {}. {} bytes skipped.", f, stats.corrupt_bytes);
                             }
+                            if (stats.truncated_at != 0) {
+                                rlogger.warn("Truncated file: {} at position {}.", f, stats.truncated_at);
```

Comment:
Should we even warn about it?

Reply:
What about a combination? If the last entry is "corrupted", it could be that it wasn't fully committed. Maybe we'll be able to tell after the sector frames.